### PR TITLE
refactor: :art: Added api for store customer review data in product detail model

### PR DESCRIPTION
**Pull request by ragulpalanisamy**
# Why?

- _Why are these changes required - if it is a new feature_
- _Issue - if the changes done is to fix some specific issue_

# Changes Done

- _Describe the changes being done_

# Testing

- _How did you test your changes ?_
- _Attach logs, screenshots (if applicable)_

# Jira Link

- _Add the link to the Story/Task for which this PR is being raised_

# References (if any)

- _Documentation Link_
- _Google Doc Links_
- _Other PRs_

### CODE REVIEW

File: client/src/components/Checkout/OrderReview.tsx [hunk 1]
```diff
@@ -1,33 +1,81 @@
...
-export interface IProductReview {
-  product: IProductReviewProps[];
-}
+import { callApi } from "../../util/callApi";
+import Textarea from "../Form/Textarea";
+
+import getStars from "../getStars";
 
 export default function (props: any) {
   const { product } = props;
```
Comment (AbdulWadudh) [inline client/src/components/Checkout/OrderReview.tsx:11]: Make this as arrow funcation component and add the name to the component

File: client/src/components/Checkout/OrderReview.tsx [hunk 2]
```diff
@@ -1,33 +1,81 @@
...
+  const productId = product[0]?.productId;
+  const customerId = product[0]?.customerId;
```
Comment (AbdulWadudh) [inline client/src/components/Checkout/OrderReview.tsx:13]: Use get() method from lodash to access the values

Comment (AbdulWadudh) [reply client/src/components/Checkout/OrderReview.tsx:13]: or why product is comming in array just send the product object

File: client/src/components/Checkout/OrderReview.tsx [hunk 3]
```diff
@@ -1,33 +1,81 @@
...
+      url: "/customerInquiry/review",
+      data: productReviewData,
+      method: "post",
+      passUserData: true,
+    });
+    if (data.status) {
+      showSuccessToast("Successfully add review");
+      window.location.reload();
+    } else {
+      showErrorToast("something went wrong");
```
Comment (AbdulWadudh) [inline client/src/components/Checkout/OrderReview.tsx:34]: Don;t add englist text replace it with appropirate korean text
and why window.location.reload is required

File: server/prisma/schema.prisma [hunk 4]
```diff
@@ -847,8 +848,8 @@ model product_review {
   image             String?   @db.VarChar
   created_at        DateTime? @default(now()) @db.Timestamptz(6)
   updated_at        DateTime? @default(now()) @db.Timestamptz(6)
-  customer          customer  @relation(fields: [customer_id], references: [customer_id], onDelete: SetDefault, onUpdate: NoAction, map: "FK_PRODUCT_REVIEW")
-  product           product   @relation(fields: [product_id], references: [product_id], onDelete: SetDefault, onUpdate: NoAction, map: "FK_PRODUCT_REVIEW_PRODUCT")
+  customer          customer  @relation(fields: [customer_id], references: [customer_id], onDelete: SetNull, onUpdate: NoAction, map: "FK_PRODUCT_REVIEW")
+  product           product   @relation(fields: [product_id], references: [product_id], onDelete: SetNull, onUpdate: NoAction, map: "FK_PRODUCT_REVIEW_PRODUCT")
```
Comment (AbdulWadudh) [inline server/prisma/schema.prisma:852]: Why from line 782 SetDefault is change to SetNull?

**Review by AbdulWadudh — CHANGES_REQUESTED** [review]

File: client/src/components/Checkout/OrderReview.tsx (hunk 1 above)
Comment (ragulpalanisamy) [reply client/src/components/Checkout/OrderReview.tsx]: Changes Done

File: client/src/components/Checkout/OrderReview.tsx (hunk 2 above)
Comment (ragulpalanisamy) [reply client/src/components/Checkout/OrderReview.tsx]: changes Done

File: client/src/components/Checkout/OrderReview.tsx (hunk 3 above)
Comment (ragulpalanisamy) [reply client/src/components/Checkout/OrderReview.tsx]: renamed the toast done

File: server/prisma/schema.prisma (hunk 4 above)
Comment (ragulpalanisamy) [reply server/prisma/schema.prisma]: while taking the db pull it automatically changed bro i didn't change but now i changed and committed the changes

**Review by AbdulWadudh — APPROVED** [review]

**Pull request merged**